specs/online_order_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'minitest/autorun'
+require 'minitest/reporters'
+require 'minitest/skip_dsl'
+
+# TODO: uncomment the next line once you start wave 3
+# require_relative '../lib/online_order'
+
+# Because an OnlineOrder is a kind of Order, and we've
+# already tested a bunch of functionality on Order,
+# we effectively get all that testing for free! Here we'll
+# only test things that are different.
+
+xdescribe "OnlineOrder" do
+  describe "#initialize" do
+    it "Is a kind of Order" do
+      # Check that an OnlineOrder is in fact a kind of Order
+
+      # Instatiate your OnlineOrder here
+      # online_order =
+      # online_order.must_be_kind_of Grocery::Order
+    end
+
+    it "Can access Customer object" do
+      # TODO: Your test code here!
+    end
+
+    it "Can access the online order status" do
+      # TODO: Your test code here!
+    end
+  end
+
+  describe "#total" do
+    it "Adds a shipping fee" do
+      # TODO: Your test code here!
+    end
+
+    it "Doesn't add a shipping fee if there are no products" do
+      # TODO: Your test code here!
+    end
+  end
+
+  describe "#add_product" do
+    it "Does not permit action for processing, shipped or completed statuses" do
+      # TODO: Your test code here!
+    end
+
+    it "Permits action for pending and paid satuses" do
+      # TODO: Your test code here!
+    end
+  end
+
+  describe "OnlineOrder.all" do
+    it "Returns an array of all online orders" do
+      # TODO: Your test code here!
+      # Useful checks might include:
+      #   - OnlineOrder.all returns an array
+      #   - Everything in the array is an Order
+      #   - The number of orders is correct
+      #   - The customer is present
+      #   - The status is present
+      # Feel free to split this into multiple tests if needed
+    end
+  end
+end
```

Comment:
Might be good to add an explicit not about how we don't need to exercise functionality we inherit directly from `Order`, since we've already tested it.